kernel/thread/thread.c

```diff
@@ -40,6 +40,8 @@ also using their queue library verbatim (sys/queue.h).
 
 */
 
+#define THD_KERNEL_STACK_SIZE (64 * 1024)
+
```

Comment:
This seems like it should be in some place where it can be shared with the use of the same value here: https://github.com/KallistiOS/KallistiOS/blob/395d99a9421aed367d7d277065080301c1a1d54d/kernel/arch/dreamcast/kernel/mm.c#L50 . To me it's arch specific and should live in the arch.h along with mem-top and such.

An alternative would be to say that KOS guarantees a 64k stack and we leave this in thread.h to share back with the arch-specific `mm`. I don't think that makes as much sense though as for something like GBA you'd likely want much less space (would be 1/5th the total ram).